internal/mage/sdk/nodejs.go

```diff
@@ -39,7 +43,26 @@ func (t NodeJS) Test(ctx context.Context) error {
 
 // Generate re-generates the SDK API
 func (t NodeJS) Generate(ctx context.Context) error {
-	panic("FIXME")
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	return util.WithDevEngine(ctx, c, func(ctx context.Context, c *dagger.Client) error {
+		generated, err := util.GoBase(c).
+			WithMountedFile("/usr/local/bin/cloak", util.DaggerBinary(c)).
+			Exec(dagger.ContainerExecOpts{
+				Args:                          []string{"cloak", "client-gen", "--lang", "nodejs", "-o", nodejsGeneratedAPIPath},
+				ExperimentalPrivilegedNesting: true,
+			}).
+			File(nodejsGeneratedAPIPath).
+			Contents(ctx)
+		if err != nil {
```

Comment:
We need to run the linter after generation, see `python` for an example.

Go doesn't do it because the Go generator lints the code on its own.

Reply:
Also -- linter must be plugged in to double check the generated code is up to date. See python/go

[🚫 blocker -- unless we do this, then the SDK will break as soon as we make API changes]